tar/storage/storage_write.c

```diff
@@ -414,7 +420,11 @@ callback_write_file_response(void * cookie,
 	switch (packetbuf[0]) {
 	case 0:
 		/* This write operation is no longer pending. */
-		C->S->nbytespending -= C->flen;
+		if (C->S->nbytespending[C->conn] < C->flen) {
```

Comment:
This is probably a good idea?

I mean, other than random memory corruption, I can't think of why this might occur.  Now, if we have random corruption, then this check could be useful... but likely a whole bunch of other things would break anyway.  *shrug*